hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -463,6 +463,9 @@ object DataSourceWriteOptions {
 
   val RECONCILE_SCHEMA: ConfigProperty[Boolean] = HoodieCommonConfig.RECONCILE_SCHEMA
 
+  val MERGE_INTO_WRITES = "hoodie.merge.into.writes"
```

Comment:
this can go to HoodieSparkSqlWriter L89 just below `CANONICALIZE_NULLABLE`, which is another internal config